nautobot/dcim/api/serializers.py

```diff
@@ -651,38 +692,43 @@ class DeviceNAPALMSerializer(serializers.Serializer):
 
 
 class ConsoleServerPortSerializer(
-    NautobotModelSerializer,
+    ModularDeviceComponentSerializerMixin,
     TaggedModelSerializerMixin,
     CableTerminationModelSerializerMixin,
     PathEndpointModelSerializerMixin,
+    NautobotModelSerializer,
 ):
     type = ChoiceField(choices=ConsolePortTypeChoices, allow_blank=True, required=False)
 
     class Meta:
         model = ConsoleServerPort
         fields = "__all__"
         extra_kwargs = {"cable": {"read_only": True}}
+        validators = []
```

Comment:
Not blocking, but it might be nice to move these three lines into `ModularDeviceComponentSerializerMixin.Meta` and have all of these component serializer Metas just inherit from it.